content/technotes/2018-07-17-phylogram.md

```diff
@@ -0,0 +1,250 @@
+---
+slug: "phylogram"
+title: "phylogram: dendrograms for evolutionary analysis"
+package_version: 2.1
+authors:
+  - name: Shaun Wilkinson
+    url: https://github.com/shaunpwilkinson
+date: 2018-07-17
+categories: technote
+topicid: 1175
```

Comment:
omit the `1175` from topicid. I'll add that right before publication to enable commenting